module/motion/HeadController/src/HeadController.cpp

```diff
@@ -60,7 +60,7 @@ namespace module::motion {
         , max_pitch(0.0)
```

Comment:
Not this line, but lines above:
Check if some of these are needed or not. Might not need `utility/math/comparison`.